docs/api/config.md

```diff
@@ -1,99 +1,152 @@
-
-### `beforeBuildCommand`
-
-Type: `string`<br/>
-Default: `""`
+## `ctx`
 
-A command to run before starting Tauri in build mode.
+<Properties rows={[
+  { property: "debug", type: "boolean", description: `Tells the CLI to build on debug mode instead of release. Can also be used as <code>tauri build --debug</code>.` },
+  { property: "exitOnPanic", type: "boolean", description: `Whether or not the webview should be killed if some Rust code error happened. Can also be used as <code>tauri dev --exit-on-panic</code> or <code>tauri dev -e</code>.` }
+]} />
 
-## ctx
-
-```js
+```js title=Example
 "ctx" {
-  "prod": false,
-  "dev": false,
-  "target": "",
   "debug": false,
-  "targetName": "",
   "exitOnPanic": false,
 }
 ```
 
-### `prod`
-
-Type: `boolean`<br/>
-Default:
-
-### `dev`
-
-Type: `boolean`<br/>
-Default:
-
-### `target`
-
-Type: `string`<br/>
-Default:
-
-### `debug`
-
-Type: `boolean`<br/>
-Default:
-
-### `targetName`
+## `tauri`
 
-Type: `string`<br/>
-Default:
-
-### `exitOnPanic`
-
-Type: `boolean`<br/>
-Default:
+<Properties rows={[
+  {
+    property: "embeddedServer", type: "object",
+    child: <Properties rows={[{ property: "active", type: "boolean", description: `Set it to <code>false</code> if you plan to serve your application statically.` }]} />
+  },
+  {
+    property: "bundle", type: "object",
+    child: <Properties rows={[
+      { property: "active", type: "boolean", description: `` },
+      { property: "targets", type: "string", description: `The name of the built application. If this is not present, then it will use the name value from your Cargo.toml file.` },
+      { property: "identifier", type: "string", description: `A string that uniquely identifies your application, in reverse-DNS form (for example, "com.example.appname" or "io.github.username.project"). For OS X and iOS, this is used as the bundle's CFBundleIdentifier value; for Windows, this is hashed to create an application GUID.` },
+      { property: "icon", type: "string[]", description: `A list of (relative to src-tauri) icon paths to use for your application bundle.` },
+      { property: "resources", type: "string[]", description: `A list of files or directories which will be copied to the resources section of the bundle. Globs are supported.` },
+      { property: "externalBin", type: "string[]", description: `A list of—either absolute or relative—paths to binaries to embed with your application.` },
+      { property: "copyright", type: "string", description: `A copyright string associated with your application.` },
+      { property: "category", type: "string", description: `What kind of application this is. This can be a human-readable string (e.g. "Puzzle game"), or a MacOS X LSApplicationCategoryType value (e.g. "public.app-category.puzzle-games"), or a GNOME desktop file category name (e.g. "LogicGame"), and cargo-bundle will automatically convert as needed for different platforms.` },
```

Comment:
cargo-bundle should be something like "Tauri bundler"

Reply:
maybe we should document the possible values:

```
pub enum AppCategory {
  Business,
  DeveloperTool,
  Education,
  Entertainment,
  Finance,
  Game,
  ActionGame,
  AdventureGame,
  ArcadeGame,
  BoardGame,
  CardGame,
  CasinoGame,
  DiceGame,
  EducationalGame,
  FamilyGame,
  KidsGame,
  MusicGame,
  PuzzleGame,
  RacingGame,
  RolePlayingGame,
  SimulationGame,
  SportsGame,
  StrategyGame,
  TriviaGame,
  WordGame,
  GraphicsAndDesign,
  HealthcareAndFitness,
  Lifestyle,
  Medical,
  Music,
  News,
  Photography,
  Productivity,
  Reference,
  SocialNetworking,
  Sports,
  Travel,
  Utility,
  Video,
  Weather,
}
```

Reply:
If there's no other possibility than a choice among this list, yup.